### ISSUE TITLE
Unnecessary backtick characters in the code block

### ISSUE THREAD

**Issue by maksim77** [MSzturc/obsidian-advanced-slides#159]
**Describe the bug**
If there are two `$` characters on the same line in the code block, then the ` character is added before each of them

**Samples to Reproduce**
1. Create new obsidian vault
2. Install Advanced Slides
3. Create туц note with this content and enable slide preview:
<pre>
```
db.books.find({$or:[{year:{$gte: 2020}},{author: "Stephen Edwin King"}]})
$some text here$
```
</pre>

4 On the slide it will be presented as:
<pre>
db.books.find({`$or:[{year:{$`gte: 2020}},{author: "Stephen Edwin King"}]})
`$some text here$`
</pre>

It's strange that in reveal itself.js as in highlight.js there is no such problem. I don't use them directly, but I checked them with minimal examples.

**Comment by MSzturc** [comment]
Had to reopen it. The change causes issues with inline latex like: 

```
You can also do inline math like $s^{-2}_{n}\sum_{i=1}^{n}$
```

**Comment by maksim77** [comment]
Yeah, I figured out the problem. I would be happy to help you with it, but unfortunately I don't understand well enough how plugins work yet...
